dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/SystemController.java

```diff
@@ -237,7 +239,11 @@ public void getTaskJsonByUid( @PathVariable( "jobType" ) String jobType, @PathVa
         setNoStore( response );
         response.setContentType( CONTENT_TYPE_JSON );
 
-        renderService.toJson( response.getOutputStream(), notifications );
```

Comment:
Does this problem apply to all three GET tasks endpoint also?

Reply:
Yes, I believe so, but I'm not sure if it matters on them.

Reply:
If we need to reverse all, I think it is actually better to do this in a lower level/service layer, since the data structure it's based on is LIFO (old→new).
If there is only one place in the UI this needs FIFO ordering, it might be better to change the UI.